cpp/daal/include/services/internal/daal_kernel_defines.h

```diff
@@ -50,13 +52,16 @@ case cpuType:
     break;                                                                                      \
 }
 
+#define DAAL_KERNEL_BUILD_MAX_INSTRACTION_SET_ID                daal::sse2
```

Comment:
Probably we should add 
```
#undef DAAL_KERNEL_BUILD_MAX_INSTRACTION_SET_ID
```
above of this line to eliminate required order of ISA extensions block's order in this file.